enki/PhysicalEngine.cpp

```diff
@@ -1167,6 +1176,9 @@ namespace Enki
 			// collide objects with walls and physics step
 			for (ObjectsIterator i = objects.begin(); i != objects.end(); ++i)
 			{
+				if(SkipPhysicsObjectsList.find(*i) != SkipPhysicsObjectsList.end())
```

Comment:
Enki's coding convention is to put a space between `if` and the opening parenthesis: `if (`.